src/daemon/win_system-info.c

```diff
@@ -0,0 +1,310 @@
+#include "win_system-info.h"
```

Comment:
This needs to be the license not the include

Reply:
Done, thank you!